src/indev/lv_indev.c

```diff
@@ -257,15 +259,15 @@ void lv_indev_read(lv_indev_t * indev_p)
 
 void lv_indev_enable(lv_indev_t * indev, bool en)
 {
-    uint8_t enable = en ? 0 : 1;
+    const uint8_t enable = en ? 1U : 0U;
```

Comment:
It feels like there's no need to introduce a new variable.
```c
void lv_indev_enable(lv_indev_t * indev, bool enable)
{
    if(indev) {
        indev->enabled = enable;
    }
    else {
        lv_indev_t * i = lv_indev_get_next(NULL);
        while(i) {
            i->enabled = enable;
            i = lv_indev_get_next(i);
        }
    }
}
```

Reply:
Yep, I tried to keep the original code as is, but your suggestion feels better to me.

Reply:
Done in 97bd64390